concent_api/core/views.py

```diff
@@ -1241,6 +1209,604 @@ def store_message_and_message_status(
+            )
+            store_message_inbox(
+                response_type       = PendingResponse.ResponseType.AckReportComputedTask,
+                client_public_key   = subtask.provider.public_key_bytes,
+                queue               = PendingResponse.PendingResponseType.Receive,
+                subtask             = subtask,
+            )
+            store_message_inbox(
+                response_type       = PendingResponse.ResponseType.VerdictReportComputedTask,
+                client_public_key   = subtask.requestor.public_key_bytes,
+                queue               = PendingResponse.PendingResponseType.ReceiveOutOfBand,
+                subtask             = subtask,
+            )
+        elif subtask.state == Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name:  # pylint: disable=no-member
+            update_subtask_state(
+                subtask                 = subtask,
+                state                   = Subtask.SubtaskState.FAILED.name,  # pylint: disable=no-member
+            )
+            store_message_inbox(
+                response_type       = PendingResponse.ResponseType.ForceGetTaskResultFailed,
+                client_public_key   = subtask.requestor.public_key_bytes,
+                queue               = PendingResponse.PendingResponseType.Receive,
+                subtask             = subtask,
+            )
+        elif subtask.state == Subtask.SubtaskState.FORCING_ACCEPTANCE.name:  # pylint: disable=no-member
+            update_subtask_state(
+                subtask                 = subtask,
+                state                   = Subtask.SubtaskState.ACCEPTED.name,  # pylint: disable=no-member
+            )
+            store_message_inbox(
+                response_type       = PendingResponse.ResponseType.ForceSubtaskResultsSettled,
+                client_public_key   = subtask.provider.public_key_bytes,
+                queue               = PendingResponse.PendingResponseType.Receive,
+                subtask             = subtask,
+            )
+            store_message_inbox(
+                response_type       = PendingResponse.ResponseType.ForceSubtaskResultsSettled,
+                client_public_key   = subtask.requestor.public_key_bytes,
+                queue               = PendingResponse.PendingResponseType.ReceiveOutOfBand,
+                subtask             = subtask,
+            )
+        # TODO when last use case will be implemented
+        # elif subtask.state == Subtask.SubtaskState.ADDITIONAL_VERIFICATION:
+
+
+def update_subtask_state(
+    subtask,
+    state,
+):
+    subtask.state = state
+    subtask.full_clean()
+    subtask.save()
+
+
+def store_message_inbox(
+    response_type       = None,
+    client_public_key   = None,
+    queue               = None,
+    subtask             = None,
+    payment_message     = None,
+):
+    client = Client.objects.get_or_create_full_clean(client_public_key)
+    if payment_message is not None:
```

Comment:
You should : 
- create PendingResponse object,
- if `payment_message` is not None, create and store PaymentCommittedMessage, then add it to PendingResponse,
- store PendingResponse.